docs-samples/data-science/semantic-link-samples/great_expectations_tutorial.ipynb

```diff
@@ -0,0 +1,2642 @@
+          ]
+        },
+        {
+          "data": {},
+          "execution_count": 1,
+          "metadata": {},
+          "output_type": "execute_result"
+        },
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Warning: PySpark kernel has been restarted to use updated packages.\n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "# install libraries\n",
+        "%pip install semantic-link great-expectations great_expectations_experimental great_expectations_zipcode_expectations\n",
+        "\n",
+        "# load %%dax cell magic\n",
+        "%load_ext sempy"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "b2eeffb5-e381-4c0d-93b7-06c94f6e80da",
+      "metadata": {},
+      "source": [
+        "2. Perform necessary imports of modules that you'll need later: "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "id": "9fb2ad54-6b22-4180-8959-43f47ed36e58",
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.livy.statement-meta+json": {
+              "execution_finish_time": "2023-10-06T15:55:33.7157053Z",
+              "execution_start_time": "2023-10-06T15:55:20.2311689Z",
+              "livy_statement_state": "available",
+              "parent_msg_id": "cbc66743-01a2-420d-9d73-5f4a021aa5b0",
+              "queued_time": "2023-10-06T15:50:38.473789Z",
+              "session_id": "efb436d1-09b2-43ab-a318-5b84c484989f",
+              "session_start_time": null,
+              "spark_jobs": {
+                "jobs": [],
+                "limit": 20,
+                "numbers": {
+                  "FAILED": 0,
+                  "RUNNING": 0,
+                  "SUCCEEDED": 0,
+                  "UNKNOWN": 0
+                },
+                "rule": "ALL_DESC"
+              },
+              "spark_pool": null,
+              "state": "finished",
+              "statement_id": 12
+            },
+            "text/plain": [
+              "StatementMeta(, efb436d1-09b2-43ab-a318-5b84c484989f, 12, Finished, Available)"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "/nfs4/pyenv-7053fd41-3592-4d02-afa4-78ae2593d477/lib/python3.10/site-packages/fuzzywuzzy/fuzz.py:11: UserWarning: Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning\n",
+            "  warnings.warn('Using slow pure-python SequenceMatcher. Install python-Levenshtein to remove this warning')\n",
```

Comment:
Is there a way to hide all the warnings from the notebook so its cleaner for the user to read?

Reply:
feel free to install the additional package at the beginning